dashboard/views.py

```diff
@@ -211,7 +211,7 @@ def grade_distribution(request, course_id=0):
     logger.info(grade_distribution.__name__)
 
     current_user = request.user.get_username()
-    grade_score_sql = "select current_grade,(select current_grade from user where sis_name=" \
+    grade_score_sql = "select distinct sis_name,current_grade,(select distinct current_grade from user where sis_name=" \
```

Comment:
This is fine but we should be adding these as unique values. I don't think we're doing this correctly. This is another ticket though.

Reply:
I can open up another ticket. But In theory, I shouldn't be doing this as there should not be duplicate records.

Reply:
I removed it from this commit, if the duplicate record issue is resolved then this can be used as a test case for testing it